test/module/irohad/synchronizer/synchronizer_test.cpp

```diff
@@ -310,3 +311,125 @@ TEST_F(SynchronizerTest, RetrieveBlockTwoFailures) {
 
   ASSERT_TRUE(wrapper.validate());
 }
+
+/**
+ * @given synchronizer receives a commit with the block peer voted for
```

Comment:
"given...receives" does not sound quite right, maybe rephrase and move "receives" part to "when"? Same in other test cases.